src/main/java/net/sf/jabref/gui/autocompleter/AutoCompleteSupport.java

```diff
@@ -233,9 +234,9 @@ private void postProcessTextChange() {
         }
 
         String text = textComp.getText();
-        E[] candidates = autoCompleter.complete(text);
+        List<E> candidates = autoCompleter.complete(text);
         renderer.update(candidates);
-        if (textComp.isEnabled() && (candidates != null) && (candidates.length > 0)) {
+        if (textComp.isEnabled() && (candidates != null) && (candidates.size() > 0)) {
```

Comment:
Normally one shouldn't return null as a list, right? As far as I see, only empty lists are returned. Should I remove the null check or leave it to be on the safe side?